### ISSUE TITLE
Translating it into Korean

### ISSUE THREAD

**Issue by taejs** [cfsamson/book-exploring-async-basics#30]
Hi, I'm Tae from Korea.
This work is so informative thanks for sharing.
If you don't mind, I'd like to translate it into Korean and share this broadly.
Following #28, I'll link your original source.

**Comment by cfsamson** [comment]
Hi. That's cool. If you post the link here too once you're done I can link to the different translations in the introduction, so readers know they have the option of reading in they native language if they wish.

**Comment by Johnny4Fun** [comment]
After working for 5 chapters, I'm wondering whether we need to create issues for per chapter. Here are the reasons:
- Easy for archiving.
- Different languages can share the same issue for one chapter.
- One issue can be too long when time goes by.
What do u guys think? @cfsamson  @taejs

**Comment by cfsamson** [comment]
Good suggestion @Johnny4Fun. Sounds good to me.

**Comment by Johnny4Fun** [comment]
Just like #31 
BTW, @cfsamson, would you change the structure or ordering of the book in the future? Considering that, I omit the chapter number.

**Comment by cfsamson** [comment]
@Johnny4Fun I don't think so. I only plan to try to keep it up to date and do minor improvements based on reported issues, at least for now.
